backend/src/openbeheer/clients.py

```diff
@@ -0,0 +1,35 @@
+from functools import cache
+from django.core.exceptions import ImproperlyConfigured
+from django.db.models.signals import post_save, post_delete
+from django.dispatch import receiver
+from django.utils.translation import gettext_lazy as _
+
+from ape_pie import APIClient
+from zgw_consumers.client import build_client
+from zgw_consumers.constants import APITypes
+from zgw_consumers.models import Service
+
+
+@cache
+def zrc_client() -> APIClient:
+    try:
+        client = build_client(
+            Service.objects.get(api_type=APITypes.zrc),
+        )
+    except Service.DoesNotExist:
+        raise ImproperlyConfigured(_("No ZRC service configured")) from None
+    # passing as arg to build_client doesn't work
+    client.headers["Accept-Crs"] = "EPSG:4326"
+    return client
```

Comment:
It is not clear to me why we need the ZRC client. I thought this would be an app to manage data in the catalogues (informatieobjecttypen, zaaktypen, ... objecttypen(?))? But not zaken, since this is the role of ZAC (Zaak Afhandel Component)?